src/ucp/wireup/wireup.c

```diff
@@ -1351,23 +1352,39 @@ ucs_status_t ucp_wireup_init_lanes(ucp_ep_h ep, unsigned ep_init_flags,
     if ((ep->cfg_index != UCP_WORKER_CFG_INDEX_NULL) &&
         /* reconfiguration is allowed for CM flow */
         !ucp_ep_has_cm_lane(ep)) {
```

Comment:
so, it seems redundant now since it is a part of `can_reconfigure` which is set outside of the func